File: common/src/main/java/dev/number6/comprehend/results/PresentableKeyPhrasesResults.java
package dev.number6.comprehend.results;

import java.time.LocalDate;
import java.util.Map;

public class PresentableKeyPhrasesResults extends ComprehensionResults<Map<String, Long>> {

    public PresentableKeyPhrasesResults(LocalDate comprehensionDate, Map<String, Long> keyPhrasesResults, String channelName) {
        super(comprehensionDate, keyPhrasesResults, channelName);
    }

    @Override
    public String toString() {
        return "KeyPhrasesResults{" +
                "comprehensionDate=" + getComprehensionDate() +
                ", keyPhrasesResults=" + getResults() +
                ", channelName='" + getChannelName() + '\'' +
                '}';
    }
}

File: common/src/main/kotlin/dev/number6/comprehend/results/PresentableKeyPhrasesResults.kt
package dev.number6.comprehend.results

import java.time.LocalDate

class PresentableKeyPhrasesResults(comprehensionDate: LocalDate,
                                   keyPhrasesResults: Map<String, Long>,
                                   channelName: String) :
        ComprehensionResults<Map<String, Long>>(comprehensionDate, keyPhrasesResults, channelName) {
    override fun toString(): String {
        return "KeyPhrasesResults{" +
                "comprehensionDate=" + comprehensionDate +
                ", keyPhrasesResults=" + results +
                ", channelName='" + channelName + '\'' +
                '}'
    }
}